### ISSUE TITLE
Thank you 🙂

### ISSUE THREAD

**Issue by joeriddles** [sbdchd/celery-types#35]
https://github.com/sbdchd/celery-types/blob/e1750be8b706ff2bc3f7471d0ed0e5f53272af7d/typings/celery/app/__init__.pyi#L13

The args in this function declaration helped me catch an issue. I was using:

```python
@celery.shared_task(
  exponential_backoff = 5
)
def some_func(...):
```

I thought `exponential_backoff` was a celery decorator param... turns out it's not! This python package is very helpful because now my type linter caught that issue. Thanks!

**Comment by jeking3** [comment]
I totally agree with the sentiment here.  This library is of great help, and the author is very responsive to bug reports.
